### PR TITLE
billing plan sdk update

### PR DESCRIPTION
**Pull request by Only2125**
https://github.com/Azure/azure-rest-api-specs/pull/7206

Reservations RP Billing Plan SDK change.

### CODE REVIEW

**Comment by AlexGhiondea** [comment]
@Only2125 could you add a PR description?

**Review by markcowl — CHANGES_REQUESTED** [review]
Please add the following to thsi PR:
(1) In the description, a link to your PR in rest-api-specs
(2) The metadata file generated by generate.ps1 ahould be updated here: `eng\mgmt\mgmtmetadata\<rpName>.txt`

**Comment by Only2125** [comment]
> Please add the following to thsi PR:
> (1) In the description, a link to your PR in rest-api-specs
> (2) The metadata file generated by generate.ps1 ahould be updated here: eng\mgmt\mgmtmetadata\<rpName>.txt

updated the metadata file and linked the swagger PR in the discription

File: eng/mgmt/mgmtmetadata/resources_resource-manager.txt [hunk 1]
```diff
@@ -12,3 +12,15 @@ Commit:      d547b7794cad660fd3cceee790e407a4d534308c
 AutoRest information
 Requested version: latest
 Bootstrapper version:    autorest@2.0.4283
+
```
Comment (markcowl) [inline eng/mgmt/mgmtmetadata/resources_resource-manager.txt]: Why do you have changes to this file and the reservations metadata file at the end?

Comment (markcowl) [reply eng/mgmt/mgmtmetadata/resources_resource-manager.txt]: this looks like a mistake

File: src/SDKs/_metadata/reservations_resource-manager.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+2019-10-01 22:33:21 UTC
```
Comment (markcowl) [inline src/SDKs/_metadata/reservations_resource-manager.txt]: this needs to be in the metadata location where you edited the other file

**Review by markcowl — CHANGES_REQUESTED** [review]

File: eng/mgmt/mgmtmetadata/resources_resource-manager.txt (hunk 1 above)
Comment (Only2125) [reply eng/mgmt/mgmtmetadata/resources_resource-manager.txt]: Sorry for the mistake. revert the change

File: eng/mgmt/mgmtmetadata/resources_resource-manager.txt [hunk 3]
```diff
@@ -11,4 +11,4 @@ Branch:      master
 Commit:      d547b7794cad660fd3cceee790e407a4d534308c
 AutoRest information
 Requested version: latest
-Bootstrapper version:    autorest@2.0.4283
+Bootstrapper version:    autorest@2.0.4283
```
Comment (markcowl) [inline eng/mgmt/mgmtmetadata/resources_resource-manager.txt]: There is no reason to update this file.  Please do not update it.

File: src/SDKs/_metadata/reservations_resource-manager.txt (hunk 2 above)
Comment (markcowl) [inline src/SDKs/_metadata/reservations_resource-manager.txt]: Please remove this file.

**Review by markcowl — DISMISSED** [review]

**Comment by isra-fel** [comment]
Hey @Only2125 
If you take a look at the changes in eng/mgmt/mgmtmetadata/reservations_resource-manager.txt you'll know its format is completely changed.
I suppose you were using the old `generate.cmd` to generate the SDK, but you really need to use `generate.ps1`.
So please regenerate your SDK code with `generate.ps1`. We expect the txt to have 3 lines changed. For example: https://github.com/Azure/azure-sdk-for-net/pull/7795/files#diff-50aabe71ddf9112b9d32088c493b59b8

Thanks

**Comment by Only2125** [comment]
> Hey @Only2125
> If you take a look at the changes in eng/mgmt/mgmtmetadata/reservations_resource-manager.txt you'll know its format is completely changed.
> I suppose you were using the old generate.cmd to generate the SDK, but you really need to use generate.ps1.
> So please regenerate your SDK code with generate.ps1. We expect the txt to have 3 lines changed. For example: https://github.com/Azure/azure-sdk-for-net/pull/7795/files#diff-50aabe71ddf9112b9d32088c493b59b8
> Thanks

Hi @isra-fel

> Hey @Only2125
> If you take a look at the changes in eng/mgmt/mgmtmetadata/reservations_resource-manager.txt you'll know its format is completely changed.
> I suppose you were using the old generate.cmd to generate the SDK, but you really need to use generate.ps1.
> So please regenerate your SDK code with generate.ps1. We expect the txt to have 3 lines changed. For example: https://github.com/Azure/azure-sdk-for-net/pull/7795/files#diff-50aabe71ddf9112b9d32088c493b59b8
> Thanks

Hi there,
I just published a new iteration, generate the file by using generate.ps1. Please take a look. Let me know 
Thanks

**Comment by isra-fel** [comment]
@Only2125 did you just close the issue by accident?

**Comment by Only2125** [comment]
Yes. I may click close after replyed

**Review by isra-fel — APPROVED** [review]

**Pull request merged**